package/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Should we rather export the modules as subobjects? Just as an idea, maybe for a future PR